src/util/AbstractAPI.js

```diff
@@ -107,12 +107,18 @@ Ext.define('Jarvus.util.AbstractAPI', {
                 if (response.aborted === true) {
                     Ext.callback(options.abort, options.scope, [response]);
                 } else if (response.status == 401 || response.statusText.indexOf('Unauthorized') !== -1) {
-
+                    
```

Comment:
@jtwiest please fix:
- Whatever editor you're using that saves trailing whitespace
- Whatever commit staging tools/process you're using that lets you not notice adding a bunch of trailing whitespace
- Whatever workflow for opening a PR leads to not looking over the Files changed view and correcting this